build/Helix/runtests.cmd

```diff
@@ -3,10 +3,10 @@ robocopy %HELIX_CORRELATION_PAYLOAD% . /s /NP
 reg add HKLM\Software\Policies\Microsoft\Windows\Appx /v AllowAllTrustedApps /t REG_DWORD /d 1 /f
 
 cd scripts
-powershell -ExecutionPolicy Bypass .\InstallTestAppDependencies.ps1
+powershell -ExecutionPolicy Bypass %1
 cd ..
 
-te MUXControls.Test.dll MUXControlsTestApp.appx IXMPTestApp.appx /enablewttlogging /unicodeOutput:false /sessionTimeout:0:15 /testtimeout:0:10 /screenCaptureOnError %* 
+te %~2 /enablewttlogging /unicodeOutput:false /sessionTimeout:0:15 /testtimeout:0:10 /screenCaptureOnError %3 
```

Comment:
This will work, but one down-side is that it limits the options that I can pass in to runtests.cmd, since anything after the 3rd param will be ignored.